gap/perm/largebase.gi

```diff
@@ -507,6 +513,8 @@ FindHomMethodsPerm.LargeBasePrimitive :=
     res := RECOG.AllJellyfish(grp);
     if res = fail then
         return TemporaryFailure;
+    elif res = false then
+        return NeverApplicable;
```

Comment:
How about changing `RECOG.AllJellyfish` to return `TemporaryFailure` resp. `NeverApplicable`, too? IMHO that'd make it easier to understand the code. Then above could become this:
```suggestion
    if not IsList(res) then
        return res;
```

Reply:
Done.